missions/shared/glossary/apollo

```diff
@@ -143,6 +143,10 @@
         "summary": "Environmental control system",
         "type": "abbreviation"
     },
+    "EDS": {
+        "summary": "Emergency detection system",
```

Comment:
You can add a `description` key as well, to provide more detailed info (short paragraph).